README.md

```diff
@@ -112,6 +112,14 @@ Docusaurus keeps track of the list of versions for the site in the `website/vers
 
 #### Cutting a new version
 
+##### After RC
```

Comment:
Theoretically we can also add examples which target not-yet-RC versions using the same steps, but in practice Snack won't be able to use them yet.